osu.Game/Graphics/UserInterface/KeyCounterMouse.cs

```diff
@@ -15,18 +18,25 @@ public KeyCounterMouse(string name, MouseButton button) : base(name)
         {
             Button = button;
         }
+        private Bindable<bool> mouseButtonsDisabled;
+
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            mouseButtonsDisabled = config.GetBindable<bool>(OsuConfig.MouseDisableButtons);

```

Comment:
I was against handling it in KeyCounterMouse myself, but I didn't see any other way at the time. I'll see if I can get to something better in the future, thanks for the hint.